ctru-rs/src/services/cfgu.rs

```diff
@@ -0,0 +1,227 @@
+    /// # Errors
+    ///
+    /// This function will return Err if there was an error initializing the
+    /// CFGU service.
+    ///
+    /// ctrulib services are reference counted, so this function may be called
+    /// as many times as desired and the service will not exit until all
+    /// instances of Fs drop out of scope.
+    pub fn init() -> crate::Result<Cfgu> {
+        unsafe {
+            let r = ctru_sys::cfguInit();
+            if r < 0 {
+                Err(r.into())
+            } else {
+                Ok(Cfgu(()))
+            }
+        }
+    }
+
+    /// Gets system region from secure info
+    pub fn get_region(&self) -> crate::Result<Region> {
+        let mut region: u8 = 0;
+        let region_pointer: *mut u8 = &mut region;
+
+        unsafe {
+            let r = ctru_sys::CFGU_SecureInfoGetRegion(region_pointer);
+            if r < 0 {
+                Err(r.into())
+            } else {
+                // The system shouldn't give an invalid value
+                Ok(Region::try_from(region).unwrap())
+            }
+        }
+    }
+
+    /// Gets system's model
+    pub fn get_model(&self) -> crate::Result<SystemModel> {
+        let mut model: u8 = 0;
+        let model_pointer: *mut u8 = &mut model;
+
+        unsafe {
+            let r = ctru_sys::CFGU_GetSystemModel(model_pointer);
+            if r < 0 {
+                Err(r.into())
+            } else {
+                // The system shouldn't give an invalid value
+                Ok(SystemModel::try_from(model).unwrap())
+            }
+        }
+    }
+
+    /// Gets system's language
+    pub fn get_language(&self) -> crate::Result<Language> {
+        let mut language: u8 = 0;
+        let language_pointer: *mut u8 = &mut language;
+
+        unsafe {
+            let r = ctru_sys::CFGU_GetSystemLanguage(language_pointer);
+            if r < 0 {
+                Err(r.into())
+            } else {
+                // The system shouldn't give an invalid value
+                Ok(Language::try_from(language).unwrap())
+            }
+        }
+    }
+
+    /// Checks if NFC is supported by the console
+    pub fn is_nfc_supported(&self) -> crate::Result<bool> {
+        let mut supported: bool = false;
+        let supported_pointer: *mut bool = &mut supported;
+
+        unsafe {
+            let r = ctru_sys::CFGU_IsNFCSupported(supported_pointer);
+            if r < 0 {
+                Err(r.into())
+            } else {
+                Ok(supported)
+            }
+        }
+    }
+
+    /// Check if the console is from the 2DS family (2DS, New2DS, New2DSXL)
+    pub fn is_2ds_family(&self) -> crate::Result<bool> {
+        let mut is_2ds_family: u8 = 0;
+        let is_2ds_family_pointer: *mut u8 = &mut is_2ds_family;
+
+        unsafe {
+            let r = ctru_sys::CFGU_GetModelNintendo2DS(is_2ds_family_pointer);
+            if r < 0 {
+                Err(r.into())
+            } else {
+                Ok(is_2ds_family < 1)
```

Comment:
The documentation is explicit that 0 equals yes, so change this to the following:
```suggestion
                Ok(is_2ds_family == 0)
```